dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/MetadataDumper.java

```diff
@@ -292,9 +293,32 @@ public <T> T runChildTask(Task<T> task) throws MetadataDumperUsageException {
                         + "* Metadata has been saved to " + outputFile + "\n"
                         + "**********************************************************"
                 );
+            } else if (connector instanceof AbstractSummaryHolderJdbcConnector) {
+                AbstractSummaryHolderJdbcConnector.Summary summary =
+                    ((AbstractSummaryHolderJdbcConnector) connector).getSummary();
+                long totalRows = summary.rowCount();
+                if (totalRows > 0) {
+                    System.out.println(
+                        "**********************************************************\n"
+                            + "* Logs has been saved to " + outputFile + "\n"
+                            + "* Earliest Row Found at " + summary.earliestInterval().getStart()
```

Comment:
I would suggest to use an explicit formatter (e.g. `DateTimeFormatter.ISO_OFFSET_DATE_TIME.withZone(ZoneOffset.UTC)`) for outputting timestamps. What formatter is used by default?